server/src/main.rs

```diff
@@ -77,6 +77,13 @@ async fn start_server() -> Server {
             .service(get_tasks_request)
```

Comment:
```suggestion
            .service(get_task_request)
```
Should there be two get_tasks_request here?